modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -120,17 +123,45 @@ storage:
           type=ethernet
           interface-name=ens2
           [ipv4]
-          address1=10.10.10.10/24,10.10.10.1
+          address1=10.10.10.10/24
           dns=8.8.8.8;
           dns-search=
           may-fail=false
           method=manual
 ----
 
+NMstate
```

Comment:
The `.` on the front makes the formatting nicer. Also suggest adding YAML:

```suggestion
.NMstate YAML
```

I would apply this to all other instances in the file.